helpers/prepare-commit-msg.sh

```diff
@@ -76,4 +76,4 @@ else
 fi
 
 # In all other cases, use commitizen to author commit message
-(exec < /dev/tty && git cz --hook)
+(exec < /dev/tty && wa-commitizen)
```

Comment:
As per [code comment in cz-cli](https://github.com/commitizen/cz-cli/blob/26533fcff90544a4152009ad38f4f15f96e7fefd/src/git/commit.js#L17-L19) we should provide the `--hook` option.